docs/management/admin/automated-response-actions.asciidoc

```diff
@@ -14,13 +14,13 @@ Add {elastic-defend}'s <<response-actions,response actions>> to detection rules
 * Automated response actions require an https://www.elastic.co/pricing[Enterprise subscription].
 * Hosts must have {agent} installed with the {elastic-defend} integration.
 * Your user role must have the ability to create detection rules and the <<endpoint-management-req,privilege>> to perform specific response actions (for example, the **Host Isolation** privilege to isolate hosts).
-* You can only add automated response actions to custom query rules.
+* You can only add automated response actions to <<create-custom-rule,custom query>>, <<create-eql-rule,event correlation (EQL)>>, and <<create-esql-rule,{esql}>> type rules.
```

Comment:
In the end we introduced this also to new-terms rule. Could you update this too? Sorry for late info! Thank you 🙇

Reply:
No problem, added in https://github.com/elastic/security-docs/pull/5797/commits/99f7c431aa060d2a912e1e8c8d72b6966a4c3e54. Thanks for the heads-up!